modules/apps/headless/headless-admin-list-type/headless-admin-list-type-impl/src/main/java/com/liferay/headless/admin/list/type/internal/resource/v1_0/ListTypeDefinitionResourceImpl.java

```diff
@@ -140,6 +152,26 @@ public ListTypeDefinition putListTypeDefinition(
 					listTypeDefinition.getName_i18n())));
 	}
 
+	@Override
+	public ListTypeDefinition putListTypeDefinitionByExternalReferenceCode(
+			String listTypeDefinitionExternalReferenceCode,
+			ListTypeDefinition listTypeDefinition)
+		throws Exception {
+
+		com.liferay.list.type.model.ListTypeDefinition oldListTypeDefinition =
```

Comment:
I think it's better to rename this variable to serviceBuilderListTypeDefinition to follow the same pattern of method `ObjectDefinitionResourceImpl#putObjectDefinitionByExternalReferenceCode`

Reply:
Sure!
Thanks for the tip!